whalesong/managers/status_v3.py

```diff
@@ -0,0 +1,114 @@
+from typing import Any, Dict, List, Optional
+from dirty_models import ModelField, HashMapField, ArrayField, IntegerField, StringIdField
+
+from . import BaseModelManager, BaseCollectionManager
+from .contact import Contact, ContactManager
+from ..models import BaseModel, DateTimeField
+from ..driver import BaseWhalesongDriver
+from ..results import Result, IteratorResult
+
+class StatusV3(BaseModel):
+    """
+    StatusV3 model
+    """
+
+    unread_count = IntegerField()
+    """
+    Unread statuses
+    """
+
+    expire_ts = DateTimeField(alias=['expireTs'])
```

Comment:
You don't need to define alias, in this case. All fields creat automatically an alias of name in camelCase.